noop.go

```diff
@@ -39,3 +39,5 @@ func (*noopSpan) Finish() {}
 func (*noopSpan) FinishedWithDuration(duration time.Duration) {}
 
 func (*noopSpan) Flush() {}
+
+func (*noopSpan) IsNoop() bool { return true }
```

Comment:
IsNoop is a convenient function to verify whether a span is `noop` or not and don't run expensive parsing when the span is noop.